Refresh Wave documentation (#516)
seqeralabs/wave: docs/index.mdx

Wave solves this problem by provisioning containers on-demand during the pipeline execution. This allows the delivery of container images that are defined precisely depending on the requirements of each pipeline task in terms of dependencies and platform architecture. This process is completely transparent and fully automated, removing all the plumbing and friction commonly needed to create, upload, and maintain dozens of container images that might be required by a pipeline execution.

To get started with Wave:

1. See the [Get started][started] guide.
1. Learn about [Nextflow integration][nf].
1. Learn about the [Wave CLI][cli].

:::note

> Wave is also available as hosted service on [Seqera Platform](https://cloud.seqera.io/). For Seqera Enterprise customers, a licensed self-hosted Wave solution is also available. Contact us [contact us](https://seqera.io/contact-us/) for more information.

Author: @ewels, this came over from the older version of the docs _as is_. I'll have to see what the reasoning might have been for this.

> Wave is also available as hosted service on [Seqera Platform](https://cloud.seqera.io/). For Seqera Enterprise customers, a licensed self-hosted Wave solution is also available. Contact us [contact us](https://seqera.io/contact-us/) for more information.

Reviewer: @pditommaso - I think we need _some_ mention of the OSS here, do you agree? Or are you trying to explicitly not mention that?

Personally I'd like to be as transparent as possible and explain what the enterprise self-hosted gives you over the OSS.

:::

[started]: ./get-started.mdx
[nf]: ./nextflow.mdx
[cli]: ./cli/index.mdx

## Wave features

Wave includes the following notable features:

### Container registries

#### Private container registries

Wave integrates with [Seqera Platform credentials management][private] enabling seamless access and publishing to private registries.

[private]: ./nextflow.mdx#access-private-container-repositories

> #### Community container registry

Reviewer: ```suggestion
#### Seqera Containers - The community container registry
```

> The Wave community registry adds to the Wave containerization lifecycle a regular container registry to host image builds permanently and is accessible publicly by anyone.

Reviewer: ```suggestion
[Seqera Containers] is a free to use service operated for the community by Seqera.

It uses Wave to build images from Conda / PyPI packages on demand, either through the [web interface](https://seqera.io/containers/) or using the [Wave CLI](./cli/index.mdx) / [Nextflow integration](./nextflow.mdx).
```

> The community registry is built using [Docker Distribution][docker] and hosted on AWS infrastructure. Images are cached and served via Cloudflare CDN.

Reviewer: ```suggestion
These images are cached and hosted permanently, being served through a [Docker Distribution][docker] registry and hosted on AWS infrastructure. Images are cached and served via Cloudflare CDN.

Images are publicly accessible to anyone, using any infrastructure, as either Docker or native Singularity images. Images can be built for both `linux/aarch64` and `linux/arm64` architectures.
```

> With [Seqera containers], you can explore the Wave community registry and create new containers on-the-fly for use in your bioinformatics pipelines.

Reviewer: ```suggestion
Note that Seqera Containers does _not_ work with custom container files, augmentation or authorisation. It only provides Conda based containers.
```

[docker]: https://github.com/distribution/distribution

> [Seqera containers]: https://seqera.io/containers/

Reviewer: ```suggestion
[Seqera Containers]: https://seqera.io/containers/
```

### Augment existing containers

Wave offers a flexible approach to container image management. It allows you to [dynamically add custom layers][augment] to existing docker images, creating new images tailored to your specific needs.
Any existing container can be extended without rebuilding it. You can add user-provided content such as custom scripts and logging agents, providing greater flexibility in the container’s configuration.

[augment]: ./provisioning.mdx#container-augmentation

### Conda based containers

> Package management systems such as Conda and Bioconda simplify the installation of scientific software. However, there’s considerable friction when it comes to using those tools to deploy pipelines in cloud environments.

Reviewer: ```suggestion
Package management systems such as Conda and Bioconda simplify the installation of scientific software.
```

:::

[started]: ./get-started.mdx
[nf]: ./nextflow.mdx
[cli]: ./cli/index.mdx

## Wave features

Wave includes the following notable features:

### Container registries

#### Private container registries

Wave integrates with [Seqera Platform credentials management][private] enabling seamless access and publishing to private registries.

[private]: ./nextflow.mdx#access-private-container-repositories

#### Seqera Containers - The community container registry

[Seqera Containers] is a free to use service operated for the community by Seqera.

It uses Wave to build images from Conda / PyPI packages on demand, either through the [web interface](https://seqera.io/containers/) or using the [Wave CLI](./cli/index.mdx) / [Nextflow integration](./nextflow.mdx).

These images are cached and hosted permanently, being served through a [Docker Distribution][docker] registry and hosted on AWS infrastructure. Images are cached and served via Cloudflare CDN.

> Images are publicly accessible to anyone, using any infrastructure, as either Docker or native Singularity images. Images can be built for both `linux/aarch64` and `linux/arm64` architectures.

Reviewer: ```suggestion
Images are publicly accessible to anyone for free and will be stored for at least 5 years. They can be pulled using any infrastructure (local, HPC, cloud) as Docker or native Singularity images. Images can be built for both `linux/aarch64` and `linux/arm64` architectures.
```

> Note that Seqera Containers does _not_ work with custom container files, augmentation or authorisation. It only provides Conda based containers.

Reviewer: ```suggestion
:::note
Note that Seqera Containers does _not_ work with custom container files, augmentation or authorisation. It only provides Conda based containers.
:::
```
